### PR TITLE
[byte-lite] Add new port (v0.2.0)

### PR DESCRIPTION
**Pull request by martinmoene**
See also issue #1489

### CODE REVIEW

File: ports/byte-lite/portfile.cmake [hunk 1]
```diff
@@ -0,0 +1,11 @@
+include(vcpkg_common_functions)
+
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO martinmoene/byte-lite
+    REF v0.2.0
+    SHA512 eefc730a39453dfc367a4478a84d4825cd85721d3c332a57321d0a5c3040a4d45921603ff24220f968dd21df61acea856ae30db8bae6c1e835a1755fb03c84b7
+)
+
+file(INSTALL ${SOURCE_PATH}/include/nonstd/byte.hpp DESTINATION ${CURRENT_PACKAGES_DIR}/include/nonstd)
```
Comment (Neumann-A) [inline ports/byte-lite/portfile.cmake]: remove this line and do a proper install.
missing vcpkg_configure_cmake, vcpkg_install_cmake and vcpkg_fixup_cmake_targets
your CMakeLists.txt looks like it should be done this way!

**Comment by ras0219-msft** [comment]
Thanks for the PR!

We'd like to also have the cmake `-config` files installed, so could you use the helpers:
```
vcpkg_configure_cmake(SOURCE_PATH ${SOURCE_PATH} PREFER_NINJA)
vcpkg_install_cmake()
vcpkg_fixup_cmake(CONFIG_PATH lib/cmake/${PORT})
```

**Comment by martinmoene** [comment]
@ras0219 Can you give a hint where is `${PORT}` is documented?

**Comment by grdowns** [comment]
Thanks for all these port contributions, @martinmoene! Due to the number of contributions in such a short span I'll thank you here and just press the merge button for the other applicable ones. :tada:

**Comment by martinmoene** [comment]
Thanks @grdowns !
Five more PRs to follow ;)

**Pull request merged**